pkg/securityscan/scanMetricsHandler.go

```diff
@@ -6,6 +6,9 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+
```

Comment:
Formatting here, delete the extra lines and goimport

Reply:
done